### PR TITLE
Fixes 1682 - missing from docs

### PR DESCRIPTION
**Pull request by jamestharpe**
Adds documentation for `getShortestPathPlansTo` and `getSimplePathPlansTo`.

### CODE REVIEW

File: docs/packages/xstate-test/index.md [hunk 1]
```diff
@@ -173,6 +173,29 @@ const plans = todosModel.getShortestPathPlans({
...
 
+### `testModel.getShortestPathPlansTo(options?)`
+
+Returns an array of testing plans based on the shortest paths from the test model's initial state to a given state specified by the `stateValue`.
+
+**Options**
+
+| Argument | Type     | Description                                                                                                    |
+| -------- | -------- | -------------------------------------------------------------------------------------------------------------- |
+| `stateValue` | string or function | Specifies a target state or a filter function which takes in the `state` and returns `true` if the state should be targeted |
```
Comment (jamesholbert) [inline docs/packages/xstate-test/index.md:184]: Thoughts on...

```
| `targetState` | string or function | Specifies a target state or a filter function which iterates over each path and returns `true` if the final state of the path matches the targetState string or filter"
```

My suggestion is by no means flawless.

Whatever change we make here should be reflected below.

File: docs/packages/xstate-test/index.md [hunk 2]
```diff
@@ -183,6 +206,29 @@ Returns an array of testing plans based on the simple paths from the test model'
...
+```js
+const todosModel = createModel(todosMachine).withEvents({
+  /* ... */
+});
+
+const plans = todosModel.getSimplePathPlansTo({
+  filter: (state) => state.matches('TODO.DELETE')
+});
+
+// Equivelent:
```
Comment (davidkpiano) [inline docs/packages/xstate-test/index.md:228]:
```suggestion
// Equivalent:
```

File: docs/packages/xstate-test/index.md [hunk 3]
```diff
@@ -173,6 +173,29 @@ const plans = todosModel.getShortestPathPlans({
...
+```js
+const todosModel = createModel(todosMachine).withEvents({
+  /* ... */
+});
+
+const plans = todosModel.getShortestPathPlansTo({
+  filter: (state) => state.matches('TODO.DELETE')
+});
+
+// Equivelent:
```
Comment (davidkpiano) [inline docs/packages/xstate-test/index.md:195]:
```suggestion
// Equivalent:
```

File: docs/packages/xstate-test/index.md (hunk 1 above)
Comment (davidkpiano) [reply docs/packages/xstate-test/index.md:184]: The current text is fine 👍

**Comment by jamestharpe** [comment]
Please LMK if anything else is needed.

**Comment by davidkpiano** [comment]
> Please LMK if anything else is needed.

This looks good, thanks! One quick fix though: these belong in `packages/xstate-test/README.md`, not in the docs - a build step will move the READMEs to the docs.

**Comment by davidkpiano** [comment]
The docs have moved to https://github.com/statelyai/docs and we will be addressing suggestions & improvements there.

**Pull request closed without merging**